packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Wysiwyg/WysiwygNav.tsx

```diff
@@ -48,199 +51,425 @@ interface WysiwygNavProps {
-                  <BulletList />
-                </IconButton>
-                <IconButton
-                  onClick={() => onActionClick('NumberList', editorRef, handleTogglePopover)}
-                  label="NumberList"
-                  name="NumberList"
-                >
-                  <NumberList />
-                </IconButton>
-              </IconButtonGroupMargin>
-              <IconButtonGroup>
-                <IconButton
-                  onClick={() => onActionClick('Code', editorRef, handleTogglePopover)}
-                  label="Code"
-                  name="Code"
-                >
-                  <Code />
-                </IconButton>
-                <IconButton
-                  onClick={() => {
-                    handleTogglePopover();
-                    onToggleMediaLib();
-                  }}
-                  label="Image"
-                  name="Image"
-                >
-                  <Image />
-                </IconButton>
-                <IconButton
-                  onClick={() => onActionClick('Link', editorRef, handleTogglePopover)}
-                  label="Link"
-                  name="Link"
-                >
-                  <Link />
-                </IconButton>
-                <IconButton
-                  onClick={() => onActionClick('Quote', editorRef, handleTogglePopover)}
-                  label="Quote"
-                  name="Quote"
-                >
-                  <Quotes />
-                </IconButton>
-              </IconButtonGroup>
+          <IconButton
+            disabled={isDisabled}
+            onClick={() => handleActionClick('Strikethrough', editorRef)}
+            label={formatMessage({
+              id: 'components.Blocks.modifiers.strikethrough',
+              defaultMessage: 'Strikethrough',
+            })}
+            name={formatMessage({
+              id: 'components.Blocks.modifiers.strikethrough',
+              defaultMessage: 'Strikethrough',
+            })}
+          >
+            <StrikeThrough />
+          </IconButton>
+        </IconButtonGroup>
+      ),
+      menu: (
+        <>
+          <Menu.Item onSelect={() => handleActionClick('Bold', editorRef)} disabled={isDisabled}>
+            <Flex tag="span" gap={2}>
+              <Bold aria-hidden fill="neutral600" />
+              {formatMessage({
+                id: 'components.Blocks.modifiers.bold',
+                defaultMessage: 'Bold',
+              })}
             </Flex>
-          </Popover.Content>
-        </Popover.Root>
-      </Flex>
+          </Menu.Item>
+          <Menu.Item onSelect={() => handleActionClick('Italic', editorRef)} disabled={isDisabled}>
+            <Flex tag="span" gap={2}>
+              <Italic aria-hidden fill="neutral600" />
+              {formatMessage({
+                id: 'components.Blocks.modifiers.italic',
+                defaultMessage: 'Italic',
+              })}
+            </Flex>
+          </Menu.Item>
```

Comment:
Menu.Item doesn't have a `startIcon` prop which I think would be very useful. But @remidej blocks will need to do this exact same thing. We could either update the DS to let MenuItem handle a startIcon, create a shared MoreMenutItem component here in the CM, or just handle it like this. wdyt?

Reply:
Ideally we'd have that in the design system, I agree. Menu.Items does accept a startIcon already, but only if it's a link, not a button like here. Links and buttons seem to be separate in the DS implementation, which is odd.

If you wanna fix it there, that would be great, otherwise I think this is fine. I don't really like the local shared abstraction though

Reply:
Ok sounds good I'll take a look in the DS but we can keep moving forward like this.